trolo/utils/assets.py

```diff
@@ -35,6 +35,8 @@ def download_model(model_name: str, output_dir: str = ".") -> Optional[str]:
     output_dir = os.path.expanduser(output_dir)
     os.makedirs(output_dir, exist_ok=True)
 
+    model_name = model_name.replace("models/", "")
```

Comment:
What is this for? We probably don't want to infer model name if user inputs a path like some/path/models/model.pth right?